tensorflow/python/keras/callbacks.py

```diff
@@ -1460,6 +1460,13 @@ class TensorBoard(Callback):
   You can find more information about TensorBoard
   [here](https://www.tensorflow.org/get_started/summaries_and_tensorboard).
 
+  Example:
+  ```python
+  tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir="./logs")
+  model.fit(x_train, y_train, epochs=2, callbacks=[tensorboard_callback])
+  #run the tensorboard command to view the visualizations
```

Comment:
Hello, it looks like you're adding an example for `tf.keras.callbacks.TensorBoard` callback instead of `tf.math.add` which is the title of this pull request. Please change the title to the corresponding contribution, and good luck on this task btw!

Reply:
Looks like 2 of my branches got mixed up! I'm working on it.